apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,66 @@
+import { Client } from "auth0";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let ACCESS_TOKEN: string;
+
+  beforeEach(function () {
+    cy.loginByAuth0Api();
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt")
+      .then((token) => {
+        ACCESS_TOKEN = token;
+      });
+  });
+
+  it("returns client info", () => {
+    const clientId: string = Cypress.env("AUTH0_CLIENT_ID") as string;
+
+    cy.request({
+      method: "GET",
+      url: `http://localhost:${PORT}/v1/clients/${clientId}`,
+      headers: {
+        Authorization: `Bearer ${ACCESS_TOKEN}`,
+      },
+    }).then((response: Cypress.Response<{ message: string }>) => {
+      expect(response.status).to.eq(200);
+      expect(response.body).property("client_id").to.eq(clientId);
+    });
+  });
+
+  it("creates new client", () => {
```

Comment:
creates and deletes a client

Reply:
I'm not sure if I should have been clearer with this feedback. I was only commenting on that the string description of the tests should be better. 

Tests should be fully isolated. Right now I believe the second to tests rely on the first test to run. If you did an it.only on the 3rd test it would fail. It looks like we should probably combine all 3 of these tests, because we have to create and delete a client for each of them.